r/vignettes/dataset.Rmd

```diff
@@ -28,13 +28,22 @@ the dataset from there. For now, datasets need to be on your local file system.
 To download the files,
 
 ```r
-bucket <- "https://ursa-labs-taxi-data.s3.us-east-2.amazonaws.com"
-dir.create("nyc-taxi")
+if(!dir.exists("nyc-taxi")) {
+  dir.create("nyc-taxi")
+}
+
 for (year in 2009:2019) {
+  if(!dir.exists(glue::glue("nyc-taxi/{year}/"))) {
```

Comment:
`glue` isn't an `arrow` dependency so I wouldn't want to add it here just for this. `file.path()` is fine.